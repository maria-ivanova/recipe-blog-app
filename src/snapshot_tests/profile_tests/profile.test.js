import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import Profile from '../../components/profile.js';

jest.mock('../../components/notifications.js', () => 'Notifications');

describe('Profile Component', () => {
    it('should render page layout', () => {
        const tree = renderer.create(
            <Routing value={{ displayName: "test", email: "test@test.com", uid: "vMmkqnOVIzQ2PjtoBZ9IkogAtt22" }}>
                <Profile />
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})