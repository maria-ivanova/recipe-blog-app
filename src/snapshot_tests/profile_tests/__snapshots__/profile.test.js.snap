// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Component should render page layout 1`] = `
<section
  className="sec content_sec"
>
  <div
    className="container tcenter"
  >
    <Notifications />
    <h2
      className="page_title"
    >
      <span>
        Профил
      </span>
    </h2>
    <form
      className="profile_form"
      onSubmit={[Function]}
    >
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Потребителско име 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          disabled={true}
          name="username"
          onChange={[Function]}
          type="text"
          value="test"
        />
      </div>
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Имейл 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          disabled={true}
          name="email"
          onChange={[Function]}
          type="email"
          value="test@test.com"
        />
      </div>
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Парола 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
      </div>
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Повтори паролата 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          name="rePassword"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
      </div>
      <div
        className="input_row"
      >
        <button
          className="btn"
          type="submit"
        >
          Редактирай
        </button>
      </div>
    </form>
  </div>
</section>
`;
